pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -1522,6 +1524,81 @@ public void testSplitTopicAndPartitionLabel() throws Exception {
         consumer2.close();
     }
 
+
+    @Test
+    public void testMetadataStoreStats() throws Exception {
```

Comment:
I wouldn't add tests to this file. It's an all-include bucket files which can easily wind up 10k lines.
I recommend moving those tests to `MetadataStoreStatsTest`

Reply:
don't worry about it, I excluded topic/consumer/producer stats:
```
        PrometheusMetricsGenerator.generate(pulsar, false, false, false, false, output);
```

Reply:
The file `PrometheusMetricsTest.java` is already very large.
I believe making it was a mistake, since creating a class that test, so many things leads to huge unreadable class.

I recommend moving those tests to `MetadataStoreStatsTest`

Reply:
I add test here because I want to ensure that all the metrics outputs as expect

Reply:
Why moving `testMetadataStoreStats` to a new class called `MetadataStoreStatsTest` will prevent you from ensuring all metrics outputs as expected?

Reply:
You resolved without answering

Reply:
sorry I missed ur comment